app/models/manageiq/providers/azure/manager_mixin.rb

```diff
@@ -25,6 +26,18 @@ def raw_connect(client_id, client_key, azure_tenant_id, subscription, proxy_uri
         $azure_log.warn("No region selected. Validating credentials against public environment.")
       end
 
+      endpoint_url = endpoint.respond_to?(:url) ? endpoint.url : endpoint.to_s
```

Comment:
`endpoint.try(:url) || endpoint.to_s` - maybe a little simpler

Reply:
True, though I'm hoping https://github.com/ManageIQ/manageiq/pull/17678 will get merged so I can just call `to_s`.

Reply:
Ok, but if ManageIQ/manageiq#17678  doesn't end up getting merged, can you simplify this line?